language/command-line-common/src/testing.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+pub const OUT_EXT: &str = "out";
```

Comment:
I'm wondering if the update baseline consts and function should also be in this file since they seem useful for testing only.